### DISCUSSION TITLE
mem overflow

### DISCUSSION THREAD

**Discussion by sevk** [rustdesk/rustdesk#9642] [General]
![ksnip_20241014-102651](https://github.com/user-attachments/assets/f5ff85e8-0a43-4c2c-a484-7a4dbfde2c42)

**Comment by 21pages** [comment]
What's the version and command line of  that process? if the version is not 1.3.1, please try https://github.com/rustdesk/rustdesk/releases/tag/1.3.1, some memleak problems are fixed.

**Reply by sevk** [reply]
yes, v1.3.0    ok, I'll try
